### DISCUSSION TITLE
undifine

### DISCUSSION THREAD

**Discussion by LuisFerVR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#458] [Q&A]
movimientos y tipo de pokemon al imprimir lo manda como indefinido. ¿Me podrían apoyar? por favor.

**Comment by ErickMontesDK** [comment]
No sé como es tu codigo, pero supongo que es porque colocas la direccion algo como data.types.0.type.name.
Los movimientos y tipos estan almacenados en vectores y  no puedes acceder a la información asi.
Puedes entrar a data.types, luego entras a la seccion 0 con types[0], o sea al elemento 0 en types.
Luego ya entras con type.name

**Reply by ErickMontesDK** [reply]
Así está en mi codigo para obtener ambos tipos

![image](https://user-images.githubusercontent.com/99063569/159101896-83f04d97-c4bf-47aa-b6bf-4f2a1c639db5.png)

**Comment by doguedogue** [comment]
```
let tipos = data.types.map(typ => typ.type.name);
let movimientos = data.moves.map(mov => mov.move.name);
```

**Comment by LuisFerVR** [comment]
Buena tarde, Gracias,  y ¿cómo lo puedes imprimir de acuerdo a tu código en
el html ?

**Comment by doguedogue** [comment]
console.log(movimientos);
console.log(tipos);

O se lo asignas a un párrafo/input/textarea.
